runtime/parachains/src/disputes.rs

```diff
@@ -1038,6 +1040,13 @@ impl<T: Config> Pallet<T> {
 			return StatementSetFilter::RemoveAll
 		}
 
+		// Reject disputes containing less votes than needed for confirmation.
+		if summary.state.validators_for.count_ones() + summary.state.validators_against.count_ones() <
+			supermajority_threshold(summary.state.validators_for.len())
```

Comment:
How does it handle duplicate votes? Shouldn't the definition of a confirmed dispute be count_ones of a union of bitvecs, not the sum of count_ones?

Reply:
Yes, I didn't thought about double voting.

Reply:
Also why does it use supermajority threshold?

Reply:
Two bugs in one line - this is a good PR! :)

Should be `byzantine_threshold`.